src/api/app/components/watchlist_component.rb

```diff
@@ -5,12 +5,17 @@ class WatchlistComponent < ApplicationComponent
     'BsRequest' => 'request'
   }.freeze
 
-  def initialize(user:, current_object:, bs_request: nil, package: nil, project: nil)
+  attr_reader :projects, :packages, :bs_requests
+
+  def initialize(user:, current_object:, bs_request: nil, package: nil, project: nil, bs_requests: [], packages: [], projects: [])
```

Comment:
What do you need the singular `bs_request`, `package` and `project` for?

Reply:
Those contain the object we are currently viewing in the app. I think they collide somehow with `current_object`, but I didn't want to fix that in this PR